x/posts/internal/simulation/operations.go

```diff
@@ -19,7 +19,7 @@ const (
 	OpWeightMsgAnswerPoll       = "op_weight_msg_answer_poll"
 	OpWeightMsgRegisterReaction = "op_weight_msg_register_reaction"
 
-	DefaultGasValue = 5000000
+	DefaultGasValue = 2000000
```

Comment:
Can we try changing this to `200000` instead? It should work

```suggestion
	DefaultGasValue = 200000
```